### DISCUSSION TITLE
绳匠们，评评七月番  以俄语遮羞的邻座艾莉同学

### DISCUSSION THREAD

**Discussion by aiTunes-dev** [share121/inter-knot#761] [其他]
![image](https://github.com/user-attachments/assets/58e17a20-a239-45f8-a77f-6b0f32e46f53)
各位绳匠们评评’‘以俄语遮羞的邻座艾莉同学’‘

**Comment by GanlinSama** [comment]
七月新番看的时候总感觉怪怪的，嘛，应该是我自己问题啦

**Comment by mixue4646** [comment]
艾莉上打不过妹妹，下打不过姐姐，除了标题是艾莉外她还有什么优势？
![Uploading 1722681559455.gif…]()

**Comment by owofile** [comment]
神！

**Comment by niegong** [comment]
人物好看😆😆

**Comment by kaijinti** [comment]
只能说不是马达马达就退坑了

**Comment by tyhpoi** [comment]
妹纸好看有趣
